src/main/java/org/broadinstitute/dsm/route/AddFamilyMemberRoute.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsm.route;
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import com.google.gson.Gson;
+import org.broadinstitute.ddp.handlers.util.Result;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.ParticipantData;
+import org.broadinstitute.dsm.db.User;
+import org.broadinstitute.dsm.model.ddp.AddFamilyMemberPayload;
+import org.broadinstitute.dsm.model.ddp.FamilyMemberDetails;
+import org.broadinstitute.dsm.security.RequestHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.utils.StringUtils;
+
+public class AddFamilyMemberRoute extends RequestHandler {
+
+    private static final Logger logger = LoggerFactory.getLogger(AddFamilyMemberRoute.class);
+
+    static final String FIELD_TYPE = "RGP_PARTICIPANTS";
```

Comment:
can you move that into FamilyMemberDetails. in case any other study will use the add family feature at one point they would have another form.

Reply:
We may do it more generic like $realm_PARTICIPANTS?